app/models/user.rb

```diff
@@ -14,12 +14,9 @@ class User < ApplicationRecord
 
   include EncryptableAttribute
 
-  encrypted_attribute_without_setter(name: :email)
-
   # IMPORTANT this comes *after* devise() call.
   include UserAccessKeyOverrides
   include UserEncryptedAttributeOverrides
-  include EmailAddressCallback
```

Comment:
maybe we inline this into `EmailAddress` model since it would become a concern only used in one place?

Reply:
This is the only place the module is used so we can/will delete it 🙂

Reply:
ope, I misread, I though it was included in two places. Yay for removing